lib/src/cli/common/target_os_type.dart

```diff
@@ -26,8 +26,24 @@ enum TargetOsType {
   windows,
 }
 
+// Cannot use Dart 2.17 enhanced enums, due to stupid issue with build_cli :-/
```

Comment:
Is there any nice way :) to explain this "stupid issue with build_cli :-/"?

Reply:
Problem is that `build_cli` will add the ctor as an option as well

Reply:
I have dropped _stupid_ for _an_.